exchanges/binance/binance.go

```diff
@@ -426,6 +430,72 @@ func (b *Binance) NewOrder(o NewOrderRequest) (NewOrderResponse, error) {
 	return resp, nil
 }
 
+// CancelOrder sends a cancel order to Binance
+// 取消订单
+func (b *Binance) CancelOrder(symbol string, orderID int64, origClientOrderID string) (CancelOrderResponse, error) {
+
+	var resp CancelOrderResponse
+
+	path := fmt.Sprintf("%s%s", apiURL, cancelOrder)
+
+	params := url.Values{}
+	params.Set("symbol", symbol)
+
+	if orderID != 0 {
+		params.Set("orderId", strconv.FormatInt(orderID, 10))
```

Comment:
looks like committed with different tab widths. I usually have my editor convert all tabs to spaces so you won't have cross platform tab differences. I notice same in more lines below.